### PR TITLE
Frontend of flix

### LINKED ISSUE #4: Create Prototype Frontend 2/2
Create friendflix.html, recsflix.html

### PR DESCRIPTION
**Pull request by ajenaration**
Frontend pages for friendflix, recsflix and footer. Additional changes to init and css.

### CODE REVIEW

**Review by Hunajpu — APPROVED** [review]
Excellent!

File: app/__init__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from flask_login import LoginManager
 
 
+from flask import Flask, escape, request, render_template
```
Comment (truongnguyenlinh) [inline app/__init__.py]: This is a duplicate of line 3

**Review by truongnguyenlinh — CHANGES_REQUESTED** [review]

File: app/templates/recsflix.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!--
```
Comment (truongnguyenlinh) [inline app/templates/recsflix.html]: Update file name

**Review by Hunajpu — APPROVED** [review]

**Review by truongnguyenlinh — APPROVED** [review]
LGTM

**Pull request merged**